diff --git a/node_modules/pkg/prelude/bootstrap.js b/node_modules/pkg/prelude/bootstrap.js
index 034bd15..90e02d7 100644
--- a/node_modules/pkg/prelude/bootstrap.js
+++ b/node_modules/pkg/prelude/bootstrap.js
@@ -74,16 +74,8 @@ if (NODE_VERSION_MAJOR < 12 || require('worker_threads').isMainThread) {
   }
 }
 
-if (process.env.PKG_EXECPATH === EXECPATH) {
-  process.argv.splice(1, 1);
-
-  if (process.argv[1] && process.argv[1] !== '-') {
-    // https://github.com/nodejs/node/blob/1a96d83a223ff9f05f7d942fb84440d323f7b596/lib/internal/bootstrap/node.js#L269
-    process.argv[1] = path.resolve(process.argv[1]);
-  }
-} else {
-  process.argv[1] = DEFAULT_ENTRYPOINT;
-}
+// always use default environment - https://github.com/vercel/pkg/issues/897#issuecomment-1663052209
+process.argv[1] = DEFAULT_ENTRYPOINT;
 
 [, ENTRYPOINT] = process.argv;
 delete process.env.PKG_EXECPATH;
